### PR TITLE
From steps also have title attribute

### PR DESCRIPTION
**Pull request by alex-mo**

### CODE REVIEW

File: templates/webform-progressbar.tpl.php [hunk 1]
```diff
@@ -43,7 +43,12 @@
           $classes[] = 'last';
         };
       ?>
-      <div class="step webform-progressbar-page <?php print implode(' ', $classes); ?>" data-form-step-number="<?php print $n; ?>">
+      <div class="step webform-progressbar-page <?php print implode(' ', $classes); ?>"
+        data-form-step-number="<?php print $n; ?>"
+      <?php if ($progressbar_pagebreak_labels): ?>
+        title="<?php print check_plain($page_labels[$n - 1]); ?>"
+      <?php endif; ?>
```
Comment (katzmo) [inline templates/webform-progressbar.tpl.php]: I think the `if` is not needed here, you want to show the title no matter if labels are set to be displayed on the progress bar or not, right?

Comment (alex-mo) [reply templates/webform-progressbar.tpl.php:50]: but they might be disabled, right?

Comment (alex-mo) [reply templates/webform-progressbar.tpl.php]: I thought the `$page_labels` will be empty in that case...

Comment (katzmo) [reply templates/webform-progressbar.tpl.php]: I’m pretty sure they’re independent: `$progressbar_pagebreak_labels` stores the setting (display page labels), the other stores the actual labels. But should be easy to test quickly?

Comment (alex-mo) [reply templates/webform-progressbar.tpl.php]: changed

**Review by katzmo — APPROVED** [review]

**Comment by alex-mo** [comment]
rebased

**Pull request merged**